cmd/wait/cmd.go

```diff
@@ -0,0 +1,161 @@
+/*
+Copyright (c) 2021 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package wait
+
+import (
+	cmv1 "github.com/openshift-online/ocm-sdk-go/clustersmgmt/v1"
+	"github.com/openshift/rosa/pkg/aws"
+	"github.com/openshift/rosa/pkg/logging"
+	"github.com/openshift/rosa/pkg/ocm"
+	"github.com/openshift/rosa/pkg/output"
+	rprtr "github.com/openshift/rosa/pkg/reporter"
+	"github.com/spf13/cobra"
+	"os"
+	"strings"
+	"time"
+)
+
+var Cmd = &cobra.Command{
+	Use:   "wait-for -c cluster -s ready -s error",
+	Short: "Waits for desired cluster states",
+	Long:  "Waits for desired cluster states, useful for CI/CD automation",
+	Run:   run,
```

Comment:
You should add an `Example` attribute here to keep it consistent with other commands